28_FFTBloom/main.cpp

```diff
@@ -709,7 +708,7 @@ class FFTBloomApp final : public examples::SimpleWindowedApplication, public app
 			// Normalization shader needs this info
 			uint16_t secondAxisFFTHalfLengthLog2 = elementsPerInvocationLog2 + workgroupSizeLog2 - 1;
 			// Create shaders
-			smart_refctd_ptr<IGPUShader> shaders[3];
+			smart_refctd_ptr<IShader> shaders[3];
```

Comment:
this example's shaders are prime candidates for combining into a single multi-entry point SPIR-V to save on compilation time